spec/integrity/i18n_spec.rb

```diff
@@ -57,4 +57,46 @@
     end
   end
 
+  describe 'English locale file' do
+    locale_files = ['config/locales', 'plugins/**/locales']
+                     .product(['server.en.yml', 'client.en.yml'])
+                     .collect { |dir, filename| Dir["#{Rails.root}/#{dir}/#{filename}"] }
+                     .flatten
+                     .map { |path| Pathname.new(path).relative_path_from(Rails.root) }
+
+    module Pluralizations
+      def self.load(path)
+        whitelist = Regexp.union([/messages.restrict_dependent_destroy/,
+                                  /js.filters.(category|new|latest|unread).title/])
```

Comment:
Small whitelist: AFAIK there's no reason to pluralize the filters and it was a nightmare to find all the places where those keys are used. :wink: